arangod/Cache/TransactionalCache.cpp

```diff
@@ -379,11 +382,12 @@ void TransactionalCache::migrateBucket(void* sourcePtr,
 std::tuple<Result, TransactionalBucket*, Table*>
 TransactionalCache::getBucket(uint32_t hash, uint64_t maxTries,
                               bool singleOperation) {
+  // allow compiler to RVO these variables
```

Comment:
RVO does not apply here, because these variables are passed as lvalues to `make_tuple` - move them instead (at least `status`).